components/automate-gateway/handler/authz.go

```diff
@@ -312,7 +309,7 @@ func (a *AuthzServer) getAllowedMap(
 		return nil, err
 	}
 	endpointMap, err := pairs.GetEndpointMapFromResponse(
-		resp.Pairs, resp.MethodsInfo, resp.MapByResourceAndAction, fullyHydrate)
+		resp.Pairs, resp.MethodsInfo, resp.MapByResourceAndAction, true)
```

Comment:
do we still need the last parameter at all?

Reply:
Not really, but I did not see any urgency in stripping that. I prefer to minimize changes when there is time pressure.

Reply:
Good eye, by the way, @bcmdarroch !